src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DAY_RENT = 40;
+  const TOTAL_RENT = DAY_RENT * days;
```

Comment:
The `TOTAL_RENT` calculation should be done after applying the discount logic. Currently, the discount is subtracted from a pre-calculated total, which might not reflect the correct discounted amount. Consider recalculating `TOTAL_RENT` after determining the applicable discount.